precompile/params.go

```diff
@@ -9,6 +9,10 @@ import "github.com/ethereum/go-ethereum/common"
 const (
 	ModifyAllowListGasCost = 20_000
 	ReadAllowListGasCost   = 5_000
+
+	MintGasCost = 20_000
```

Comment:
Only real nit on this PR is to not make minting dynamic gas. Let's use 30_000 and call it a day for all ops.

Reply:
done